src/components/BookList.spec.js

```diff
@@ -34,92 +33,31 @@ describe('Book list', () => {
   ]
 
   it('renders two listitems', () => {
-    const mockOnHandleActiveReadingStatus = jest.fn()
-
-    render(
-      <BookList
-        books={books}
-        handleActiveReadingStatus={mockOnHandleActiveReadingStatus}
-      />
-    )
+    render(<BookList books={books} readingStatus={''} />)
 
     const bookListItems = screen.getAllByRole('listitem')
     expect(bookListItems).toHaveLength(2)
   })
```

Comment:
I would expect a test, which tests if the `h2` (`<h2>{filterTitle(readingStatus)}</h2>`) is there or not. So you basically need this test three times and check for the different states.

Reply:
Ah, you are doing that! Nice 👍 So I would recommend here to check that `h2` is _not_ in the document 🙂